### PR TITLE
feat: Add document in `QuantitySelector`

### PR DESCRIPTION
**Pull request by ArthurTriis1**
Signed-off-by: Arthur Andrade <arthurfelandrade@gmail.com>

## What's the purpose of this pull request?

This PR adds storybook documentation for QuantitySelector component.

## How does it work?

Added the TokenTable and TokenRow components to make it easier to document tokens in the storybook.

## How to test it?

Run storybook and see `http://localhost:6006/?path=/docs/organisms-quantityselector--quantity-selector`.

## Checklist

<em>You may erase this after checking them all ;)</em>

- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#4](https://github.com/vtex-sites/nextjs.store/pull/4))* 

- PR description
- [x] Updated the Storybook - *if applicable*.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Add Storybook documentation for `QuantitySelector` component ([#81](https://github.com/vtex-sites/nextjs.store/pull/81))
```
Comment (filipewl) [inline CHANGELOG.md:12]:
```suggestion
- Add Storybook documentation for `QuantitySelector` component ([#85](https://github.com/vtex-sites/gatsby.store/pull/85))
```

**Review by filipewl — APPROVED** [review]
Looks good to me! Left a fix suggestion for the change log.

**Review by eduardoformiga — APPROVED** [review]
Good job! 👏

File: src/components/ui/QuantitySelector/QuantitySelector.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    name="Overview-Disabled"
+    args={{
+      disabled: true
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Usage
```
Comment (renatamottam) [inline src/components/ui/QuantitySelector/QuantitySelector.stories.mdx:74]:
```suggestion

---

## Usage
```

File: src/components/ui/QuantitySelector/QuantitySelector.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+<Meta
+  title="Molecules/QuantitySelector"
+  component={QuantitySelector}
+  argTypes={{
+    max: {
+      type: {
+        name: 'number',
+      },
+      defaultValue: 10,
+      description: "The maximum value the quantity selector can receive"
```
Comment (renatamottam) [inline src/components/ui/QuantitySelector/QuantitySelector.stories.mdx:15]: Could these descriptions be on `QuantitySelector.txs` directly? [See Toggle.tsx](https://github.com/vtex-sites/gatsby.store/blob/main/src/components/ui/Toggle/Toggle.tsx) as reference.

**Review by renatamottam — COMMENTED** [review]
My lint is accusing some fixes on this file:
https://github.com/vtex-sites/nextjs.store/pull/90/files#diff-45907629a64c0068ab581241340c8e23494e2b84e9a09699a58dc9aa2e1e259e

**Review by hellofanny — APPROVED** [review]
LGTM 🤩! It seems like there are no changes from https://github.com/vtex-sites/gatsby.store/pull/85. But if there is any, make sure to update there too. :)

**Pull request merged**